proposals/0357-alpenglow_validator_admission_ticket.md

```diff
@@ -0,0 +1,191 @@
+next epoch (bank.epoch() + 1). This is the same as now.
+
+2. Perform stake activation and deactivation so that the intended stake values
+are used for the new epoch. This operation occurs before any transactions are
+processed in the new epoch’s bank.
+
+3. The calculation iterates all vote accounts and filters those that meet
+the following criteria:
+
+  - The account has a balance of at least 1.6 SOL
+
+  - The account has a BLS public key
+
+4. If the number of filtered accounts exceeds 2,000, sort them according to the
+following rules and select the top 2,000. Otherwise, return the entire list:
+
+  - Sort in descending order of stake (largest to smallest)
+
+  - If multiple validators have exactly the same amount of stake and including
+  all of them would exceed the 2,000 limit, then exclude all of them
+
+5. Subtract 1.6 SOL from the vote account for each validator in the accepted
+list from the previous step
+
+6. Record the VAT fee subtraction in the bank, which reduces the bank’s
+capitalization. The recording of fee subtraction occurs when the bank is
+frozen even though fee reduction happens before any transaction in the
+bank is executed.
+
+## Operation Considerations
+
+- To be included in epoch e+1, validator operators must ensure the vote
+account has at least 1.6 SOL before epoch e-1 ends
+
+- Validator operators must ensure they have BLS public key specified in
+their vote account
+
+- We will not allow removing the BLS public key
+
+## Alternatives Considered
+
+- Have all validators send a transaction to deduct 1.6 SOL VAT fee every epoch.
+A few problems with this approach:
+
+  - This transaction must land before the staked validators are selected
+
+  - If this validator ends up not being selected, this VAT fee needs to be
+returned, which means we have to implement some type of accounting and ensure
+the VAT fee returning properly lands on the chain.
+
+  - It does deduct an additional transaction fee from the vote account
+
+  - If somehow validators make a mistake by sending multiple transactions,
+we also need to return the fee collected.
+
+## Impact
+
+Validators not providing BLS public key or desired fee will not be able to
+participate in an Alpenglow epoch regardless of their stake. Also, only the
+nodes selected by this process will receive votes or certificates from other
+selected validators in real time.
+
+Validator operators need to ensure they have enough funds and correct BLS
+public key before end of epoch e-1 to participate in epoch e+1. This poses some
+new operation challenges.
+
+We are requiring correct setup of vote accounts at end of epoch e-1 for
+inclusion in epoch e+1 because of the following reason: We want to be
+absolutely sure that we have a finalized state before we enter epoch e+1.
+Specifically, if the last slots of epoch e-1 are not finalized at the end
+of the epoch, we have enough slots in epoch e to finalize epoch e-1's state.
+
```

Comment:
we're beating a dead horse with this "valid vote account" thing

the material impact is that the voting set is now strictly capped at 2000. we can also mention that operators must now keep their vote account topped up instead of their autorized-voter/gossip-identity